crypto/cryptodev.c

```diff
@@ -379,11 +379,6 @@ int cryptodev_op(FAR struct csession *cse,
   int error = OK;
   uint32_t hid;
 
-  if (cop->len > 64 * 1024 - 4)
-    {
-      return -E2BIG;
-    }
-
```

Comment:
Even size_t has a limit, should we compare cop->len against (SIZE_MAX - 4) ?

Reply:
but why do we need subtract -4?